docs/rules/no-generic-link-text.md

```diff
@@ -0,0 +1,83 @@
+# No Generic Link Text
```

Comment:
This is pretty much copied from our `erblint-github` docs though the examples were changed to `.jsx` instead of `.erb`. 

The two behave nearly exactly the same so why not 🤷🏻‍♀️